RecoLocalTracker/SiPixelRecHits/src/PixelCPEBase.cc

```diff
@@ -194,7 +195,8 @@ void PixelCPEBase::fillDetParams() {
     p.bx = Bfield.x();
 
     //---  Compute the Lorentz shifts for this detector element
-    if ((theFlag_ == 0) || DoLorentz_) {  // do always for generic and if(DOLorentz) for templates
+    if ((theFlag_ == 0) || useLAFromDB_ ||
+        doLorentzFromAlignment_) {  // do always for generic and if(DOLorentz) for templates
```

Comment:
the comment about `if(DOLorentz)` does not seem to match the new implementation; please update